src/components/with-authentication/withAuthentication.jsx

```diff
@@ -1,91 +0,0 @@
-import React, { Component } from 'react';
```

Comment:
I imagined that we wouldn't need this wrapper anymore since it seems like it's probably better to handle auth and all the configuration in the app itself, but I could be wrong

Reply:
Yeah, frontend-platform's initialize method does pretty much all of this for you.